app/src/main/java/com/android/sample/feature/camera/CameraUtils.kt

```diff
@@ -61,3 +67,31 @@ fun rotateBitmap(bitmap: Bitmap, rotationDegrees: Int): Bitmap {
   val rotatedBitmap = Bitmap.createBitmap(bitmap, 0, 0, bitmap.width, bitmap.height, matrix, true)
   return rotatedBitmap
 }
+
+/**
+ * Function to open the gallery to select an image
+ *
+ * @param photoPickerLauncher ManagedActivityResultLauncher<PickVisualMediaRequest, Uri?>
+ */
+fun openGallery(photoPickerLauncher: ManagedActivityResultLauncher<PickVisualMediaRequest, Uri?>) {
+  photoPickerLauncher.launch(
```

Comment:
Solid work on `openGallery` and `uriToBitmap`. Here are a few quick thoughts:

1. **Error Handling in `uriToBitmap`**: The `try-catch` block is great for safety. 
_Optional_: To improve clarity, maybe log a more descriptive message on failure instead of just printing the stack trace (e.g., `"Failed to convert URI to Bitmap"`).

2. **Function Documentation**: Love the thorough comments! They make the code easy to understand. Just a small suggestion: for the `@param uri` in `uriToBitmap`, maybe add that it should be non-null to reinforce the expectation.

Overall, looks clean and efficient! 👍

Reply:
If this method will not be reused anywhere, then everything look good